packages/shared/src/utils/hexUtils.ts

```
@@ -39,11 +39,27 @@ function isHexString(value: string, length?: number): boolean {
   return utils.isHexString(addHexPrefix(value), length);
 }
 
+function hexStringToUtf8String(hexString: string): string {
+  const hex = hexString.replace('0x', '');
+
+  try {
+    const bytes = new Uint8Array(
+      hex.match(/.{1,2}/g)?.map((byte) => parseInt(byte, 16)) || [],
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)